pom.xml

```diff
@@ -22,7 +22,7 @@
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <!-- remember to change the io.jenkins.tools.bom artifact when changing this -->
-    <jenkins.version>2.387.3</jenkins.version>
+    <jenkins.version>2.433</jenkins.version>
```

Comment:
Not yet in an LTS version of course. I have no strong opinion as to whether we should wait for that or not.